pages/contribute/editorial_board_guide.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Editorial board guide
+summary: This guide is there to help editors.
+sidebar: contribute
+---
+
+## Github handiness
+
+### Working on your own feature branch
```

Comment:
@bedroesb I suggest you move this section on "Working on your feature branch" down, to its respective subsection. I suggest don't start the guide with the advance way for the editor. Some editors would follow the simple way, then the guide is more difficult to understand for them